recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
```

Comment:
Try this one:

https://pypi.io/packages/source/h/holoviews/holoviews-1.5.0.tar.gz